actions/image_partition_action.go

```diff
@@ -80,6 +75,11 @@ GPT sets the partition type to Linux Swap.
 For msdos partition types hex codes see: https://en.wikipedia.org/wiki/Partition_type
 For gpt partition type GUIDs see: https://systemd.io/DISCOVERABLE_PARTITIONS/
 
+- start -- offset from beginning of the disk there the partition starts.
```

Comment:
It'd be good to note why this is optional and what would happen if it isn't actually provided.